realm/realm-library/src/main/java/io/realm/internal/TableOrView.java

```diff
@@ -341,6 +341,14 @@
 
     TableQuery where();
 
+    /**
+     * Find a row with in the table or view with the given index.
+     *
+     * @param rowIndex The index of the row.
```

Comment:
`@param rowIndex The index of the row. ` ->
`@param rowIndex The index of the row in the source table.`